config/initializers/spree_permitted_attributes.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  module PermittedAttributes
+    ATTRIBUTES += %i[video_attributes]
+    mattr_reader *ATTRIBUTES
+
+    @@video_attributes = [:product_id, :url, :file, :tag_name, tag_names: []]
```

Comment:
why is tags not nested attribute and tag_names instead?

and why should we have both tag_name and tag_names?